simln-lib/src/lib.rs

```diff
@@ -48,6 +48,9 @@ pub enum NodeId {
     Alias(String),
 }
 
+#[derive(Serialize, Debug, Clone)]
+pub struct Address(String);
```

Comment:
I think that we can live without this type alias and just use `#[serde(with = "serializers::serde_address")]` to point to custom serialization/deserialization of a `String` to reduce LOC a little.